todo_app/tests/integration_tests/test_index_page.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+import requests
+
+from todo_app.tests.conftest import stub
+
+
+def test_index_page(monkeypatch, client):
+    # Replace requests.get(url) with our own function
+    monkeypatch.setattr(requests, "get", stub)
```

Comment:
Given this is already patched in the underlying client method, we could drop this bit here

Reply:
Ah yeah, good point!